### PR TITLE
Drop support for end-of-life Wagtail versions, add support for Wagtail 6.0

### PR DESCRIPTION
**Pull request by Stormheg**
https://endoflife.date/wagtail

This significantly reduces the number of available Python/Django/Wagtail combination we have to test. Good for the environment 🌍

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -28,16 +27,12 @@ basepython =
...
     django50: django>=5.0,<5.1
 
-    wagtail41: wagtail~=4.1
-    wagtail42: wagtail~=4.2
-    wagtail50: wagtail~=5.0
-    wagtail51: wagtail~=5.1
-    wagtail52: wagtail~=5.2
+    wagtail42: wagtail~=4.2.0
+    wagtail52: wagtail~=5.2.0
+    wagtail60: wagtail~=6.0.0
```
[the remark this answers is not in the document]
Comment (jhonatan-lopes) [reply tox.ini:35]: TIL

**Review by jhonatan-lopes — APPROVED** [review]
LGTM :rocket: thanks @Stormheg

File: tox.ini (hunk 1 above)
[the remark this answers is not in the document]
Comment (jhonatan-lopes) [reply tox.ini:35]: @Stormheg there are no 4.X versions supported anymore. I believe we should remove the 4.2 here

https://github.com/wagtail/wagtail/wiki/Release-schedule

**Comment by Stormheg** [comment]
Oops, merged without removing 4.2 from the testing matrix. I'll go and fix that now.

**Pull request merged**